src/main/java/net/glowstone/generator/structures/GlowTemple.java

```diff
@@ -21,6 +21,8 @@ public GlowTemple(World world, int chunkX, int chunkZ) {
         types.put(Biome.JUNGLE, TempleType.JUNGLE_TEMPLE);
         types.put(Biome.JUNGLE_HILLS, TempleType.JUNGLE_TEMPLE);
         types.put(Biome.SWAMPLAND, TempleType.WITCH_HUT);
+        types.put(Biome.FOREST, TempleType.IGLOO);        //TODO: Temporary, change to ICE_FLAT
+        types.put(Biome.FOREST_HILLS, TempleType.IGLOO);  //TODO: Temporary, change to TAIGA_COLD
```

Comment:
I put it like that because I couldn't find an Ice Plains biome, since there were more forest biomes... I'll change it back when I'm done testing the generation of the structure.